src/controllers/userController.ts

```diff
@@ -60,4 +64,60 @@ const findUserByEmail = async (email: string) => {
     } catch (error) {
         return null
     }
-}
\ No newline at end of file
+}
+
+export const signIn = (req: Request, res: Response) => {
+  UserModel.findOne({
+    where: {
+      email: req.body.email,
+    },
+  })
+    .then((user) => {
+      if (!user) {
+        return res.status(404).send({ message: "User Not found." });
```

Comment:
The same here, I would love it if you return the response using the API_RESPONSE to make our responses uniform